x/treasury/keeper/keeper.go

```diff
@@ -532,7 +532,8 @@ func (k *Keeper) SplitKeyringFee(ctx context.Context, from, to string, fee uint6
 		return err
 	}
 
-	zenrockFee := uint64(math.Round(float64(fee) * (float64(prms.KeyringCommission) / 100.0)))
+	zenrockFeeDec := sdkmath.LegacyNewDecFromBigInt(big.NewInt(int64(fee))).Mul(sdkmath.LegacyNewDecFromBigInt(big.NewInt(int64(prms.KeyringCommission))).Quo(sdkmath.LegacyNewDec(100)))
```

Comment:
```suggestion
	zenrockFeeDec := sdkmath.LegacyNewDecFromInt(sdkmath.NewIntFromUint64(fee)).Mul(sdkmath.LegacyNewDecFromInt(sdkmath.NewIntFromUint64(prms.KeyringCommission)).Quo(sdkmath.LegacyNewDec(100)))

```

we can directly use the uint64s like this